spec/factories/customers.rb

```diff
@@ -1,6 +1,10 @@
 FactoryGirl.define do
   factory :customer do
-    first_name "MyString"
-    last_name "MyString"
+    sequence :first_name do |n|
```

Comment:
I like this sequence method. I will adapt it to my factories to generate random data.